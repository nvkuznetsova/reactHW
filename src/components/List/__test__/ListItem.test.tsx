import React from 'react';
import { shallow } from 'enzyme';

import { ListItem } from '..';

describe('ListItem', () => {
  it('should match snapshot', () => {
    const children = <span>Test</span>;
    const component = shallow(<ListItem>{children}</ListItem>);

    expect(component).toMatchSnapshot();
  });
});