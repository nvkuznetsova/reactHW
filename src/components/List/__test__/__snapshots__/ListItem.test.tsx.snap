// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem should match snapshot 1`] = `
<li
  className="list-group-item "
>
  <span>
    Test
  </span>
</li>
`;
